ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/workshopAdmin/view/WorkshopAdminViewImpl.java

```diff
@@ -0,0 +1,122 @@
+import com.google.gwt.uibinder.client.UiFactory;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.uibinder.client.UiHandler;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import com.google.inject.assistedinject.Assisted;
+import com.sencha.gxt.core.client.Style;
+import com.sencha.gxt.data.shared.ListStore;
+import com.sencha.gxt.widget.core.client.Composite;
+import com.sencha.gxt.widget.core.client.button.TextButton;
+import com.sencha.gxt.widget.core.client.event.SelectEvent;
+import com.sencha.gxt.widget.core.client.grid.ColumnConfig;
+import com.sencha.gxt.widget.core.client.grid.ColumnModel;
+import com.sencha.gxt.widget.core.client.grid.Grid;
+import org.iplantc.de.admin.desktop.client.workshopAdmin.WorkshopAdminView;
+import org.iplantc.de.admin.desktop.client.workshopAdmin.events.DeleteMembersClickedEvent;
+import org.iplantc.de.admin.desktop.client.workshopAdmin.events.RefreshMembersClickedEvent;
+import org.iplantc.de.admin.desktop.client.workshopAdmin.events.SaveMembersClickedEvent;
+import org.iplantc.de.admin.desktop.client.workshopAdmin.model.MemberProperties;
+import org.iplantc.de.client.events.EventBus;
+import org.iplantc.de.client.models.groups.Member;
+import org.iplantc.de.collaborators.client.util.UserSearchField;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author dennis
+ */
+public class WorkshopAdminViewImpl extends Composite implements WorkshopAdminView {
+
+    interface WorkshopAdminViewImplUiBinder extends UiBinder<Widget, WorkshopAdminViewImpl> {}
+
+    @UiField(provided = true) UserSearchField userSearch;
+    @UiField TextButton deleteButton;
+    @UiField Grid<Member> grid;
+    @UiField(provided = true) ListStore<Member> listStore;
+    @UiField(provided = true) WorkshopAdminViewAppearance appearance;
+
+    private MemberProperties memberProperties;
+    private final ArrayList<HandlerRegistration> globalHandlerRegistrations = new ArrayList<>();
+
+    private static WorkshopAdminViewImplUiBinder uiBinder = GWT.create(WorkshopAdminViewImplUiBinder.class);
+
+    @Inject
+    public WorkshopAdminViewImpl(final WorkshopAdminViewAppearance appearance,
+                                 final MemberProperties memberProperties,
+                                 @Assisted ListStore<Member> listStore) {
+        this.userSearch = new UserSearchField(userSearchEventTag);
+        this.appearance = appearance;
+        this.memberProperties = memberProperties;
+        this.listStore = listStore;
+        initWidget(uiBinder.createAndBindUi(this));
+        grid.getSelectionModel().setSelectionMode(Style.SelectionMode.MULTI);
+    }
+
+    @Override
+    public <H extends EventHandler> void addGlobalEventHandler(GwtEvent.Type<H> type, H handler) {
+        HandlerRegistration registration = EventBus.getInstance().addHandler(type, handler);
+        globalHandlerRegistrations.add(registration);
+    }
```

Comment:
The only reason we needed the global event bus here was because the `UserSearchField` uses it.